.github/workflows/woke.yml

```diff
@@ -1,5 +1,5 @@
 # yamllint disable rule:line-length
-name: Check for non-inclusive language
+name: Woke
```

Comment:
The previous name is more descriptive as a task name IMHO.

Reply:
The badge is generated with tha name, so I wanted to make the badge toe have `woke` in it. Although, woke is not a widely-know tool, so maybe better have something more descriptive like `Detect non-inclusive language`

Reply:
Is this possible to let the badge have a more descriptive name so that it can be consistent with github workflow ?

Reply:
Right now, the workflow name in PR CI jobs is `Woke / Detect non-inclusive language`, and the badge name is `Woke`. I think that's good enough.

Reply:
Thanks for addressing this, I agree with the current change.